src/stability_sdk/animation_ui.py

```diff
@@ -369,6 +360,8 @@ def postprocess_video(fps: int, reverse: bool, interp_mode: str, interp_factor:
             else:
                 _, video_ext = os.path.splitext(video_to_postprocess)
                 output_video = video_to_postprocess.replace(video_ext, f"{suffix}.mp4")
+
+            yield { error_log: gr.update(label="Status", value="Compiling frames to MP4...", visible=True) }
```

Comment:
should it still be named as 'error_log'? if it is also used now as a status bar, maybe naming it as 'status_bar' or 'status_log' would be more correct

Reply:
yeah good idea